src/js/BookReader/Mode1Up.js

```diff
@@ -205,10 +283,10 @@ export class Mode1Up {
    * Note this calls drawLeafs
    */
   resizePageView() {
-    const viewWidth  = this.br.refs.$brContainer.prop('clientWidth');
-    const oldScrollTop  = this.br.refs.$brContainer.prop('scrollTop');
-    const oldPageViewHeight = this.br.refs.$brPageViewEl.height();
-    const oldPageViewWidth = this.br.refs.$brPageViewEl.width();
+    const viewWidth  = this.$scrollContainer.prop('clientWidth');
```

Comment:
Q: are we/should we be moving away from stashing Dom in `this.br.refs`?

Reply:
Anything that's not necessary to be globally accessible, should be class-specific. $brPageViewEl is Mode1Up specific, so hiding it here from the rest of the world.